### ISSUE TITLE
Getting started with Codespaces

### ISSUE THREAD

**Issue by devigned** [Azure/azure-capi-cli-extension#10]
## Actual
When I [follow the instructions to run in Codespaces](https://github.com/Azure/azure-capi-cli-extension#developer-setup), I run into an error when executing `azdev test`. Below is the output from executing the instructions from my Codespaces terminal.

## Expectation
I would expect to follow the instructions and be able to execute the project tests from the Codespaces terminal.

## Output from Codespaces

```
vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ python3 -m venv env
n -m pip install -U pip
python -m pip install -r requirements.txt
azdev setup --repo . --ext capi --verbose
vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ source env/bin/activate
(env) vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ 
(env) vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ python -m pip install -U pip
Requirement already satisfied: pip in ./env/lib/python3.6/site-packages (21.0.1)
(env) vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ python -m pip install -r requirements.txt
Requirement already satisfied: azdev in ./env/lib/python3.6/site-packages (from -r requirements.txt (line 1)) (0.1.29)
Requirement already satisfied: black in ./env/lib/python3.6/site-packages (from -r requirements.txt (line 2)) (20.8b1)
Requirement already satisfied: Jinja2 in ./env/lib/python3.6/site-packages (from -r requirements.txt (line 3)) (2.11.3)
Requirement already satisfied: MarkupSafe in ./env/lib/python3.6/site-packages (from -r requirements.txt (line 4)) (1.1.1)
Requirement already satisfied: sphinx==1.6.7 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (1.6.7)
Requirement already satisfied: flake8 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (3.9.0)
Requirement already satisfied: tox in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (3.23.0)
Requirement already satisfied: pytest-xdist in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (2.2.1)
Requirement already satisfied: pytest>=5.0.0 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (6.2.2)
Requirement already satisfied: pyyaml in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (5.4.1)
Requirement already satisfied: azure-storage-blob<2.0.0,>=1.3.1 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (1.5.0)
Requirement already satisfied: mock in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (4.0.3)
Requirement already satisfied: wheel==0.30.0 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (0.30.0)
Requirement already satisfied: isort==4.3.21 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (4.3.21)
Requirement already satisfied: docutils in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (0.16)
Requirement already satisfied: knack in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (0.7.2)
Requirement already satisfied: pylint==2.3.0 in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (2.3.0)
Requirement already satisfied: gitpython in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (3.1.14)
Requirement already satisfied: requests in ./env/lib/python3.6/site-packages (from azdev->-r requirements.txt (line 1)) (2.25.1)
Requirement already satisfied: mccabe in ./env/lib/python3.6/site-packages (from pylint==2.3.0->azdev->-r requirements.txt (line 1)) (0.6.1)
Requirement already satisfied: astroid>=2.2.0 in ./env/lib/python3.6/site-packages (from pylint==2.3.0->azdev->-r requirements.txt (line 1)) (2.5.1)
Requirement already satisfied: snowballstemmer>=1.1 in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (2.1.0)
Requirement already satisfied: Pygments>=2.0 in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (2.8.1)
Requirement already satisfied: sphinxcontrib-websupport in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (1.2.4)
Requirement already satisfied: alabaster<0.8,>=0.7 in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (0.7.12)
Requirement already satisfied: six>=1.5 in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (1.15.0)
Requirement already satisfied: setuptools in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (40.6.2)
Requirement already satisfied: babel!=2.0,>=1.3 in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (2.9.0)
Requirement already satisfied: imagesize in ./env/lib/python3.6/site-packages (from sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (1.2.0)
Requirement already satisfied: typed-ast<1.5,>=1.4.0 in ./env/lib/python3.6/site-packages (from astroid>=2.2.0->pylint==2.3.0->azdev->-r requirements.txt (line 1)) (1.4.2)
Requirement already satisfied: wrapt<1.13,>=1.11 in ./env/lib/python3.6/site-packages (from astroid>=2.2.0->pylint==2.3.0->azdev->-r requirements.txt (line 1)) (1.12.1)
Requirement already satisfied: lazy-object-proxy>=1.4.0 in ./env/lib/python3.6/site-packages (from astroid>=2.2.0->pylint==2.3.0->azdev->-r requirements.txt (line 1)) (1.5.2)
Requirement already satisfied: azure-common>=1.1.5 in ./env/lib/python3.6/site-packages (from azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (1.1.26)
Requirement already satisfied: azure-storage-common~=1.4 in ./env/lib/python3.6/site-packages (from azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (1.4.2)
Requirement already satisfied: python-dateutil in ./env/lib/python3.6/site-packages (from azure-storage-common~=1.4->azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (2.8.1)
Requirement already satisfied: cryptography in ./env/lib/python3.6/site-packages (from azure-storage-common~=1.4->azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (3.4.6)
Requirement already satisfied: pytz>=2015.7 in ./env/lib/python3.6/site-packages (from babel!=2.0,>=1.3->sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (2021.1)
Requirement already satisfied: py>=1.8.2 in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (1.10.0)
Requirement already satisfied: toml in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (0.10.2)
Requirement already satisfied: iniconfig in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (1.1.1)
Requirement already satisfied: attrs>=19.2.0 in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (20.3.0)
Requirement already satisfied: pluggy<1.0.0a1,>=0.12 in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (0.13.1)
Requirement already satisfied: importlib-metadata>=0.12 in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (3.7.3)
Requirement already satisfied: packaging in ./env/lib/python3.6/site-packages (from pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (20.9)
Requirement already satisfied: zipp>=0.5 in ./env/lib/python3.6/site-packages (from importlib-metadata>=0.12->pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (3.4.1)
Requirement already satisfied: typing-extensions>=3.6.4 in ./env/lib/python3.6/site-packages (from importlib-metadata>=0.12->pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (3.7.4.3)
Requirement already satisfied: chardet<5,>=3.0.2 in ./env/lib/python3.6/site-packages (from requests->azdev->-r requirements.txt (line 1)) (4.0.0)
Requirement already satisfied: certifi>=2017.4.17 in ./env/lib/python3.6/site-packages (from requests->azdev->-r requirements.txt (line 1)) (2020.12.5)
Requirement already satisfied: idna<3,>=2.5 in ./env/lib/python3.6/site-packages (from requests->azdev->-r requirements.txt (line 1)) (2.10)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./env/lib/python3.6/site-packages (from requests->azdev->-r requirements.txt (line 1)) (1.26.4)
Requirement already satisfied: regex>=2020.1.8 in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (2021.3.17)
Requirement already satisfied: click>=7.1.2 in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (7.1.2)
Requirement already satisfied: pathspec<1,>=0.6 in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (0.8.1)
Requirement already satisfied: mypy-extensions>=0.4.3 in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (0.4.3)
Requirement already satisfied: dataclasses>=0.6 in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (0.8)
Requirement already satisfied: appdirs in ./env/lib/python3.6/site-packages (from black->-r requirements.txt (line 2)) (1.4.4)
Requirement already satisfied: cffi>=1.12 in ./env/lib/python3.6/site-packages (from cryptography->azure-storage-common~=1.4->azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (1.14.5)
Requirement already satisfied: pycparser in ./env/lib/python3.6/site-packages (from cffi>=1.12->cryptography->azure-storage-common~=1.4->azure-storage-blob<2.0.0,>=1.3.1->azdev->-r requirements.txt (line 1)) (2.20)
Requirement already satisfied: pyflakes<2.4.0,>=2.3.0 in ./env/lib/python3.6/site-packages (from flake8->azdev->-r requirements.txt (line 1)) (2.3.0)
Requirement already satisfied: pycodestyle<2.8.0,>=2.7.0 in ./env/lib/python3.6/site-packages (from flake8->azdev->-r requirements.txt (line 1)) (2.7.0)
Requirement already satisfied: gitdb<5,>=4.0.1 in ./env/lib/python3.6/site-packages (from gitpython->azdev->-r requirements.txt (line 1)) (4.0.5)
Requirement already satisfied: smmap<4,>=3.0.1 in ./env/lib/python3.6/site-packages (from gitdb<5,>=4.0.1->gitpython->azdev->-r requirements.txt (line 1)) (3.0.5)
Requirement already satisfied: tabulate in ./env/lib/python3.6/site-packages (from knack->azdev->-r requirements.txt (line 1)) (0.8.9)
Requirement already satisfied: colorama in ./env/lib/python3.6/site-packages (from knack->azdev->-r requirements.txt (line 1)) (0.4.4)
Requirement already satisfied: jmespath in ./env/lib/python3.6/site-packages (from knack->azdev->-r requirements.txt (line 1)) (0.10.0)
Requirement already satisfied: argcomplete in ./env/lib/python3.6/site-packages (from knack->azdev->-r requirements.txt (line 1)) (1.12.2)
Requirement already satisfied: pyparsing>=2.0.2 in ./env/lib/python3.6/site-packages (from packaging->pytest>=5.0.0->azdev->-r requirements.txt (line 1)) (2.4.7)
Requirement already satisfied: pytest-forked in ./env/lib/python3.6/site-packages (from pytest-xdist->azdev->-r requirements.txt (line 1)) (1.3.0)
Requirement already satisfied: execnet>=1.1 in ./env/lib/python3.6/site-packages (from pytest-xdist->azdev->-r requirements.txt (line 1)) (1.8.0)
Requirement already satisfied: apipkg>=1.4 in ./env/lib/python3.6/site-packages (from execnet>=1.1->pytest-xdist->azdev->-r requirements.txt (line 1)) (1.5)
Requirement already satisfied: sphinxcontrib-serializinghtml in ./env/lib/python3.6/site-packages (from sphinxcontrib-websupport->sphinx==1.6.7->azdev->-r requirements.txt (line 1)) (1.1.4)
Requirement already satisfied: virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0 in ./env/lib/python3.6/site-packages (from tox->azdev->-r requirements.txt (line 1)) (20.4.3)
Requirement already satisfied: filelock>=3.0.0 in ./env/lib/python3.6/site-packages (from tox->azdev->-r requirements.txt (line 1)) (3.0.12)
Requirement already satisfied: distlib<1,>=0.3.1 in ./env/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox->azdev->-r requirements.txt (line 1)) (0.3.1)
Requirement already satisfied: importlib-resources>=1.0 in ./env/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox->azdev->-r requirements.txt (line 1)) (5.1.2)
 . --ext capi --verboseces/azure-capi-cli-extension (foo ✗) $ azdev setup --repo 
=======================
| Azure CLI Dev Setup |
=======================
Azure CLI:
    PyPI
Azure CLI extension repos:
    /workspaces/azure-capi-cli-extension
Azure CLI extensions:
    /workspaces/azure-capi-cli-extension/src/capi
 Installing packages 
=====================
Upgrading pip...
Running: /usr/local/py-utils/bin/python -m pip install --upgrade pip
Installing `azure-cli`...
Running: /usr/local/py-utils/bin/python -m pip install --upgrade azure-cli
Installing `azure-cli-testsdk`...
Running: /usr/local/py-utils/bin/python -m pip install git+https://github.com/Azure/azure-cli@master#subdirectory=src/azure-cli-testsdk
Adding extension '/workspaces/azure-capi-cli-extension/src/capi'...
Running: /usr/local/py-utils/bin/python -m pip install -e /workspaces/azure-capi-cli-extension/src/capi
Elapsed time: 0 min 58 sec
 Finished dev setup! 
=====================
(env) vscode ➜ /workspaces/azure-capi-cli-extension (foo ✗) $ azdev test
=============
| Run Tests |
=============
No module named 'azure.cli'
Traceback (most recent call last):
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/knack/cli.py", line 215, in invoke
    cmd_result = self.invocation.execute(args)
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/knack/invocation.py", line 225, in execute
    cmd_result = parsed_args.func(params)
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/knack/commands.py", line 148, in __call__
    return self.handler(*args, **kwargs)
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/knack/commands.py", line 255, in _command_handler
    result = op(client, **command_args) if client else op(**command_args)
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/azdev/operations/testtool/__init__.py", line 46, in run_tests
    path_table = get_path_table()
  File "/workspaces/azure-capi-cli-extension/env/lib/python3.6/site-packages/azdev/utilities/path.py", line 193, in get_path_table
    from azure.cli.core.extension import EXTENSIONS_DIR  # pylint: disable=import-error
ModuleNotFoundError: No module named 'azure.cli'
```

**Comment by mboersma** [comment]
Thanks @devigned! I'm confident this has been fixed by #11, so I'm closing it. But if you have a chance to try the Codespaces setup again, please let us know if you still see other issues.

**Comment by devigned** [comment]
Works for me. Thank you!
